### PR TITLE
Bug/delete moose lake from nadina/2176

### PR DESCRIPTION
**Pull request by conbrad**
- NWFC disabled moose lake in wf1 app, deleting the station in HFI makes it so their prep values are calculated
- It can be added back through station admin later if need be

# Test Links:
[Percentile Calculator](https://wps-pr-2178.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-2178.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-2178.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-2178.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-2178.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Fire Behaviour Advisory](https://wps-pr-2178.apps.silver.devops.gov.bc.ca/fire-behaviour-advisory)
[HFI Calculator](https://wps-pr-2178.apps.silver.devops.gov.bc.ca/hfi-calculator)
[FWI Calculator](https://wps-pr-2178.apps.silver.devops.gov.bc.ca/fwi-calculator)

### CODE REVIEW

**Comment by andrea-williams** [comment]
Is this PR even necessary? Why not just let NWFC do the delete through Manage Stations themselves?

**Comment by conbrad** [comment]
> Is this PR even necessary? Why not just let NWFC do the delete through Manage Stations themselves?

Yes, because it doesn't show up in HFI front since it's disabled in WFWX

**Comment by andrea-williams** [comment]
> > Is this PR even necessary? Why not just let NWFC do the delete through Manage Stations themselves?
> 
> Yes, because it doesn't show up in HFI front since it's disabled in WFWX

Ohhhh gotcha

**Review by andrea-williams — APPROVED** [review]

File: api/alembic/versions/2dac381ad6e7_remove_moose_lake_from_nadina.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+
+def get_nadina_planning_area_id(session: Session):
+    res = session.query(planning_areas_table) \
+        .filter(planning_areas_table.c.name.ilike(f'Nadina Zone%'))
+    return int(res.first().id)
+
+
+def upgrade():
+    # ### commands auto generated by Alembic ###
```
Comment (Sybrand) [inline api/alembic/versions/2dac381ad6e7_remove_moose_lake_from_nadina.py:50]: Pedantic - but these auto generated comments can be removed.

**Review by Sybrand — APPROVED** [review]
Looks good, @conbrad I'd just remove those auto-generated comments.

**Review by Sybrand — COMMENTED** [review]
Testing post merge review comment.

File: api/alembic/versions/2dac381ad6e7_remove_moose_lake_from_nadina.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm.session import Session
+from app.utils.time import get_utc_now
+
+
+# revision identifiers, used by Alembic.
+revision = '2dac381ad6e7'
+down_revision = '157ba3a3f3fb'
```
Comment (Sybrand) [inline api/alembic/versions/2dac381ad6e7_remove_moose_lake_from_nadina.py:16]: Test

**Pull request merged**